migrate.sh

```diff
@@ -116,4 +116,17 @@ done
 loginCluster $1 "$restoreClusterID"
 sh ./updateEBSVolumeTags.sh
 
+echo -e "\nDeleting the old/backup cluster"
+loginCluster $1 "$backupClusterID"
+consumers=( $(kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}') )
+for consumer in $consumers
+do
+  kubectl patch storageconsumers  ${consumer} -n openshift-storage -p '{"metadata":{"finalizers":null}}' --type=merge
+  kubectl delete storageconsumers  ${consumer} -n openshift-storage
+done
```

Comment:
We can replace this with
```suggestion
kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl patch storageconsumer -p '{"metadata":{"finalizers":null}}' --type=merge -n openshift-storage
kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl delete storageconsumer -n openshift-storage
```

Reply:
We also need to patch storageSystem and storagecluster

Reply:
Sure